Source/Extensions/Blazorise.DataGrid/Blazorise.DataGrid.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>1591</NoWarn>
+    <Version>1.1.3</Version>
```

Comment:
We control versions from a `Blazorise.props` file. You can safely remove this line.

Reply:
Removed line